features/admin.authentication-flow-builder.v1/data/flow-sequences/mfa/index.ts

```diff
@@ -19,3 +19,4 @@
 export { default as EmailOTPSecondFactorLoginSequence } from "./second-factor-email-otp-sequence.json";
 export { default as SMSOTPSecondFactorLoginSequence } from "./second-factor-sms-otp-sequence.json";
 export { default as TOTPSecondFactorLoginSequence } from "./second-factor-totp-sequence.json";
+export { default as PushAuthSecondFactorLoginSequence } from "./second-factor-push-auth-sequence.json";
```

Comment:
Have we made sure that this can be enabled/disabled conditionally in console UI?